components/EntryList/EntriesList.js

```diff
@@ -29,7 +29,7 @@ export default function EntriesList() {
       {reversedMoods.map((entry) => (
         <Styled.Container key={entry._id}>
           <Styled.AnimationContainer>
-            <Animation color={entry.color} opacity={entry.intensity} />
+            {/* <Animation color={entry.color} opacity={entry.intensity} /> */}
```

Comment:
```suggestion
```

Reply:
is already solved (simplified visuals for entries) in current main